profiles/kentik_snmp/apc/apc_ups.yml

```diff
@@ -7,6 +7,7 @@ extends:
 provider: kentik-ups
 
 sysobjectid:
+  - 1.3.6.1.4.1.935.1.1.1     # APC EasyUPS
```

Comment:
@pkanakaraj19 - this OID vendor code `1.3.6.1.4.1.935` is [owned by a different company than APC](https://oidref.com/1.3.6.1.4.1.935). Are you positive that this is an APC device?

Reply:
@thezackm that's the OID I'm seeing in SNMP walk output, and in the New Relic entity summary page (currently classified as 'kentik_default'). This is the NMC used on the device: https://www.apc.com/us/en/product/E3SOPT001/network-card-for-easy-ups-3s-3m-3l-and-galaxy-pw-2nd-gen-ups-/

Reply:
@thezackm confirmed with a person on site where the UPS is located - it's an APC Easy UPS 3M model with the above linked NMC.

Reply:
I'll connect with APC support and see if the NMC needs an update. Please hold off on the PR and thanks for pointing it out! @thezackm

Reply:
gotcha; note that a lot of times this happens when a vendor buys another company; it's definitely plausible. will mark this to hold until you get back; thanks.